index.html

```diff
@@ -2,114 +2,114 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>My Basic Site
-
-  Leave these links here...
-  But this should be in a comment...
+  <title>Kate leslie</title>
+  <!--Leave these links here...
+  But this should be in a comment-->
+  <link rel="stylesheet" href="styles/style.css">
```

Comment:
In the future, it's best practice to put the stylesheets from your app after things like google fonts or external CSS files. We'll review that tomorrow but a heads up.

This is because if you wanted to modify something in the font-awesome stylesheet you need to be either after it or more specific or both.